data.sql

```diff
@@ -7,3 +7,17 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) V
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Pikachu','2021-01-07', 1, false, 15.04);
 
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Devimon','2017-05-12', 5, true, 11);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Charmander','2020-02-8', 0, false, 11);
+
```

Comment:
- [ ] Kindly make the **"Charmander"** animal has the data like the ones given on the **[project's requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Animal%3A%20His%20name%20is%20Charmander.%20He%20was%20born%20on%20Feb%208th%2C%202020%2C%20and%20currently%20weighs%20%2D11kg.%20He%20is%20not%20neutered%20and%20he%20has%20never%20tried%20to%20escape.)** like on the `weight_kg` column. 
Once you update it make sure, you also update other `queries` required for the project to still follow the requirements.